inst/app.crunch.io/api/datasets/1/multitables/apidocs-ca-mr-tabbook.json

```diff
@@ -0,0 +1 @@
meric_value":0,"missing":false,"id":0,"name":"Not Selected"},{"numeric_value":null,"missing":true,"id":-1,"name":"No Data"}]}},"missing":false},{"id":3,"value":{"derived":false,"references":{"alias":"allpets_3","name":"Bird"},"id":"0003","type":{"ordinal":false,"class":"categorical","categories":[{"numeric_value":1,"selected":true,"id":1,"missing":false,"name":"Selected"},{"numeric_value":0,"missing":false,"id":0,"name":"Not Selected"},{"numeric_value":null,"missing":true,"id":-1,"name":"No Data"}]}},"missing":false}],"class":"enum"}},{"references":{"alias":"allpets","subreferences":[{"alias":"allpets_1","name":"Cat"},{"alias":"allpets_2","name":"Dog"},{"alias":"allpets_3","name":"Bird"}],"description":"Do you have any of these animals as pets? Please select all that apply.","name":"All pets owned"},"derived":true,"type":{"ordinal":false,"subvariables":["0001","0002","0003"],"class":"categorical","categories":[{"numeric_value":1,"selected":true,"id":1,"missing":false,"name":"Selected"},{"numeric_value":0,"missing":false,"id":0,"name":"Not Selected"},{"numeric_value":null,"missing":true,"id":-1,"name":"No Data"}]}},{"derived":true,"references":{"alias":"petloc","subreferences":[{"alias":"petloc_home","name":"Home"},{"alias":"petloc_work","name":"Work"}],"description":"Name the kinds of pets you have at these locations.","name":"Pets by location"},"type":{"subtype":{"class":"variable"},"elements":[{"id":1,"value":{"derived":false,"references":{"alias":"petloc_home","name":"Home"},"id":"0001","type":{"ordinal":false,"class":"categorical","categories":[{"numeric_value":1,"id":1,"name":"Cat","missing":false},{"numeric_value":2,"id":2,"name":"Dog","missing":false},{"numeric_value":3,"id":3,"name":"Bird","missing":false},{"numeric_value":8,"id":8,"name":"Skipped","missing":true},{"numeric_value":9,"id":9,"name":"Not Asked","missing":true}]}},"missing":false},{"id":2,"value":{"derived":false,"references":{"alias":"petloc_work","name":"Work"},"id":"0002","type":{"ordinal":false,"class":"categorical","categories":[{"numeric_value":1,"id":1,"name":"Cat","missing":false},{"numeric_value":2,"id":2,"name":"Dog","missing":false},{"numeric_value":3,"id":3,"name":"Bird","missing":false},{"numeric_value":8,"id":8,"name":"Skipped","missing":true},{"numeric_value":9,"id":9,"name":"Not Asked","missing":true}]}},"missing":false}],"class":"enum"}},{"derived":false,"references":{"alias":"petloc","description":"Name the kinds of pets you have at these locations.","name":"Pets by location","subreferences":[{"alias":"petloc_home","name":"Home"},{"alias":"petloc_work","name":"Work"}]},"type":{"ordinal":false,"subvariables":["0001","0002"],"class":"categorical","categories":[{"numeric_value":1,"id":1,"name":"Cat","missing":false},{"numeric_value":2,"id":2,"name":"Dog","missing":false},{"numeric_value":3,"id":3,"name":"Bird","missing":false},{"numeric_value":8,"id":8,"name":"Skipped","missing":true},{"numeric_value":9,"id":9,"name":"Not Asked","missing":true}]}}],"n":20}}]}]}
```

Comment:
Would you mind prettifying this JSON? (Been meaning to look into how to make `httptest` write out pretty JSON without parsing in R, but it's worth doing manually here.)